sys/kern/tty.c

```diff
@@ -179,6 +179,29 @@ tty_t *tty_alloc(void) {
   return tty;
 }
 
+void tty_free(tty_t *tty) {
+  assert(!tty_opened(tty));
+  assert(tty_detached(tty));
+  kfree(M_DEV, tty->t_line.ln_buf);
```

Comment:
It is not implemented yet, but `mtx_destroy` is a good idea.

Reply:
Done.